endpoints/remotehost/remotehost

```diff
@@ -116,14 +142,14 @@ function endpoint_remotehost_test_start() {
                 let count=$count+1
             done
 
-            local server_outside_endpoint=1
+            server_outside_endpoint=1
```

Comment:
Style.  As remotehost was almost completely re-written, I decided to document the local and globals at the top of each function.

Reply:
Ok, I was just curious because I recall seeing a problem somewhere once when `local` was used on the same line where the variable was assigned a value....I was curious if you were seeing that or not because I never really understood what was happening -- I just remember that separating the use of `local` and the assignment seemed to fix a problem.